helm/templates/arbiter/statefulset-arbiter-rs.yaml

```diff
@@ -44,6 +44,20 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
       {{- end }}
     spec:
+      initContainers:
+        - name: copy-keyfile
```

Comment:
init container should be wrapped around `{{- if eq .Values.config.security.authorization "enabled" }}`